# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ncp/head-tail.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


The usual deprecated options ...

  - Rene Rebe <rene@exactcode.de>

--- ./Makefile.vanilla	2004-05-17 00:09:17.661888280 +0200
+++ ./Makefile	2004-05-17 00:11:00.021327280 +0200
@@ -99,14 +99,14 @@
 compile: \
 warn-auto.sh conf-cc
 	( cat warn-auto.sh; \
-	echo exec "`head -1 conf-cc`" '-c $${1+"$$@"}' \
+	echo exec "`head -n 1 conf-cc`" '-c $${1+"$$@"}' \
 	) > compile
 	chmod 755 compile
 
 preprocess: \
 warn-auto.sh conf-cc
 	( cat warn-auto.sh; \
-	echo exec "`head -1 conf-cc`" '-E $${1+"$$@"}' \
+	echo exec "`head -n 1 conf-cc`" '-c $${1+"$$@"}' \
 	) > preprocess
 	chmod 755 preprocess
 
@@ -114,7 +114,7 @@
 warn-auto.sh conf-ld
 	( cat warn-auto.sh; \
 	echo 'main="$$1"; shift'; \
-	echo exec "`head -1 conf-ld`" \
+	echo exec "`head -n 1 conf-ld`" \
 	'-o "$$main" "$$main".o $${1+"$$@"}' \
 	) > load
 	chmod 755 load
@@ -131,11 +131,11 @@
 
 uint32.h: \
 preprocess tryulong.c uint32.h1 uint32.h2
-	if test `./preprocess tryulong.c | tail -1` = 32; then cat uint32.h2; else cat uint32.h1; fi > uint32.h
+	if test `./preprocess tryulong.c | tail -n 1` = 32; then cat uint32.h2; else cat uint32.h1; fi > uint32.h
 
 uint64.h: \
 preprocess tryulong.c uint64.h1 uint64.h2
-	if test `./preprocess tryulong.c | tail -1` = 64; then cat uint64.h2; else cat uint64.h1; fi > uint64.h
+	if test `./preprocess tryulong.c | tail -n 1` = 64; then cat uint64.h2; else cat uint64.h1; fi > uint64.h
 
 haveip6.h: \
 tryip6.c choose compile haveip6.h1 haveip6.h2
@@ -182,7 +182,7 @@
 
 auto_home.c: \
 auto-str conf-home
-	./auto-str auto_home `head -1 conf-home` > auto_home.c
+	./auto-str auto_home `head -n 1 conf-home` > auto_home.c
 
 # automatically generated dependancies follow
 alloc.o: \
